src/vorta/borg/_compatibility.py

```diff
@@ -0,0 +1,24 @@
+from pkg_resources import parse_version
+
+MIN_BORG_FOR_FEATURE = {
+    'REPOKEY_BLAKE2': parse_version('1.1.4'),
+    # add new version-checks here.
```

Comment:
from our docs/changes.rst:
- pre-1.1.4 potential data corruption issue
- pre-1.0.9 manifest spoofing vulnerability (CVE-2016-10099)
- pre-1.0.9 potential data loss

vorta could warn for < 1.1.4 or < 1.0.9 if you like.